### PR TITLE
Image with Placeholder

### LINKED ISSUE #397: Create Animated Image component to animate between placeholder and image from url

### PR DESCRIPTION
**Pull request by jenipharachel**
fixes #397

### CODE REVIEW

**Comment by jenipharachel** [comment]
@sangameshsomawar _a Created Animated Image component with placeholder image shown while image is fetched from URL. Please review and merge!

File: src/components/AnimatedImage.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+            opacity.value = 1;
+          }}
+          {...rest}
+        />
+
+        {!isImageLoaded && (
+          <Container position="absolute">
+            <ImagePlaceholder width={imageWidth} height={imageHeight} />
+          </Container>
+        )}
```
Comment (sangameshsomawar) [inline src/components/AnimatedImage.js:91]:
```suggestion
        {!isImageLoaded && (
          <Container position="absolute" alignItems="center" justifyContent="center" width={imageWidth} height={imageHeight}>
            <ImagePlaceholder  />
          </Container>
        )}
```

**Review by sangameshsomawar — APPROVED** [review]
@jenipharachel LGTM

https://user-images.githubusercontent.com/93119254/185043449-1912dbbd-fa2a-464d-8114-cf633936cbf1.mp4

**Comment by jenipharachel** [comment]
@sangameshsomawar Pushed the disabled state of button during loading, in this PR as well.

**Comment by sangameshsomawar** [comment]
@jenipharachel You can merge PR if all required changes are done.

**Pull request merged**